src/saveload/mod.rs

```diff
@@ -4,25 +4,25 @@
 //!
 //! This module provides two `SystemData` implementors:
 //!
-//! * `WorldSerialize` and
-//! * `WorldDeserialize`
+//! * `SerializeComponents` and
+//! * `DeserializeComponents`
 //!
-//! Fetching those makes it very easy to serialize or deserialize
-//! components. However, be aware that you cannot fetch storages
+//! Reading those makes it very easy to serialize or deserialize
+//! components. However, be aware that you cannot read storages
 //! used in `WorldDeserialize` with the same system. E.g.
 //! `type SystemData = (WorldDeserialize<'a, Marker, MyError, (Pos, Vel)>, WriteStorage<'a, Vel>)`
-//! is not valid since both `WorldDeserialize` and `WriteStorage` would fetch the same component
+//! is not valid since both `DeserializeComponents` and `WriteStorage` would fetch the same component
 //! storage mutably.
```

Comment:
Is this documentation up to date on saveload stuff?

Reply:
@WaDelma Yes, do you have reason to believe otherwise?

Reply:
Ah, so there are just renamings missing?
```
 //! used in `WorldDeserialize` with the same system. E.g.
 //! `type SystemData = (WorldDeserialize<'a, Marker, MyError, (Pos, Vel)>, WriteStorage<'a, Vel>)`
```

Also the the saveload example looks totally different.

Reply:
I think World* does not exist anymore.

Reply:
I wanted to eventually add them back but not sure now and don't have much time, I'll let you two decide.